src/macros.rs

```diff
@@ -283,47 +283,35 @@ macro_rules! assert_display_snapshot_matches {
 #[macro_export]
 macro_rules! assert_snapshot_matches {
     ($value:expr, @$snapshot:literal) => {
-        $crate::_assert_snapshot_matches!(
+        assert_snapshot_matches!(
```

Comment:
I think this needs the `$crate::` prefix.

Reply:
Because it's referring to itself, this isn't required (I think - or how would tests be passing?)

Reply:
I think this works because the test imports the macro directly. A crate could also use `insta::assert_snapshot_matches!`, in which case I believe this would fail to find the macro.

Reply:
I'm not sure what the rules are right now but I believe any macro that invokes another macro needs to have the `$crate::` prefix atm.

Reply:
The compiler uses `$crate` to expand names that are not accessible from the caller into a name that is accessible. In this case the macro is calling itself, so any caller must have access to the name. Previously, it was calling another macro, and so could have required it. 

Here's more context for anyone interested: https://github.com/rust-lang-nursery/reference/blob/master/src/macros-by-example.md#hygiene

I ran a quick test locally - started a crate, linked to this version, and it does work...

Reply:
Did the test also invoke the macro using `insta::assert_snapshot_matches!`? If you `use` the macro then the string `assert_snapshot_matches!(...)` will of course be able to find that macro.

Reply:
Good idea - let me check

Reply:
@jonas-schievink you were right! I've updated. Thanks for the feedback